packages/myst-common/src/types.ts

```diff
@@ -84,6 +84,10 @@ export type RoleData = {
   body?: ParseTypes;
 };
 
+export type DirectiveContext = {
+  parseMyST: (source: string, offset?: number) => GenericParent;
```

Comment:
```suggestion
  parseMyst: (source: string, offset?: number) => GenericParent;
```

This matches the current exported function name.

Reply:
Yeah, I think I've been internally thinking that we should rename our other types to `MyST` (I've had the same thought about DOI vs doi). However, happy to make that a future all-in-one pass if we get buy in.